MenuApp/MenuApp.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
We can have the solution at the root folder and no need to nest it inside MenuApp folder.

Reply:
modified